run_integration_test.go

```diff
@@ -720,3 +720,128 @@ func TestRunCreateOptions_Marshal(t *testing.T) {
 
 	assert.Equal(t, string(bodyBytes), expectedBody)
 }
+
+func TestRunsListForOrganization(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	apTest, _ := createAgentPool(t, client, orgTest)
+
+	wTest, _ := createWorkspaceWithOptions(t, client, orgTest, WorkspaceCreateOptions{
+		Name:          String(randomString(t)),
+		ExecutionMode: String("agent"),
+		AgentPoolID:   &apTest.ID,
+	})
+	rTest1, _ := createRun(t, client, wTest)
+	rTest2, _ := createRun(t, client, wTest)
+
+	t.Run("without list options", func(t *testing.T) {
+		rl, err := client.Runs.ListForOrganization(ctx, orgTest.Name, nil)
+		require.NoError(t, err)
+
+		found := []string{}
+		for _, r := range rl.Items {
+			found = append(found, r.ID)
+		}
+
+		assert.Contains(t, found, rTest1.ID)
+		assert.Contains(t, found, rTest2.ID)
+		assert.Equal(t, 1, rl.CurrentPage)
+		assert.Equal(t, 2, rl.TotalCount)
+	})
+
+	t.Run("without list options and include as nil", func(t *testing.T) {
+		rl, err := client.Runs.ListForOrganization(ctx, orgTest.Name, &RunListForOrganizationOptions{
+			Include: []RunIncludeOpt{},
+		})
+		require.NoError(t, err)
+		require.NotEmpty(t, rl.Items)
+
+		found := []string{}
+		for _, r := range rl.Items {
+			found = append(found, r.ID)
+		}
+
+		assert.Contains(t, found, rTest1.ID)
+		assert.Contains(t, found, rTest2.ID)
+		assert.Equal(t, 1, rl.CurrentPage)
+		assert.Equal(t, 2, rl.TotalCount)
+	})
+
+	t.Run("with list options", func(t *testing.T) {
+		t.Skip("paging not supported yet in API")
```

Comment:
I don't think this is true and pagination seems to work as expected. Does this test not produce the expected result?

Reply:
You are right, I removed this `t.Skip` and this test works as expected. I updated the PR description with new test results. Thank you!